developer-docs-site/docs/concepts/basics-accounts.md

```diff
@@ -2,42 +2,107 @@
+- Start by generating a new private key, public key pair.
+- From the user get the preferred signature scheme for the account: If the account should use a single signature or if it should require multiple signatures for signing a transaction. 
+- Combine the public key with the user's signature scheme to generate a 32-byte authentication key. 
+- Initialize the sequence number to 0. The sequence number represents the next transaction sequence number, to prevent replay attacks of transactions. Both the authentication key and the sequence number are stored in the account as an initial account resource. 
+- Create the 32-byte account address from the initial authentication key. 
 
-Each account also stores a `sequence_number`, which represents the next transaction sequence number to prevent replay attacks of transactions. This is initialized to `0` at account creation time.
+From now on, the user should use the private key for signing the transactions with this account. 
 
-## Authentication Keys
+## Signature schemes
 
-Each account stores an authentication key. This authentication key enables account owners to rotate their private key(s) associated with the account without changing the address that hosts their account. During rotation, the authentication key is updated based upon the newly-generated private, public key-pair(s).
+An account can send transactions. The Aptos Blockchain supports two signature schemes: 
 
-### Single signer authentication
+1. The [Ed25519](https://ed25519.cr.yp.to/) for single signature transactions, and 
+2. The MultiEd25519, for multi-signature transactions. 
 
-To generate an authentication key and account address:
+:::note
+The Aptos Blockchain defaults to single signature transactions.
+:::
+
+  
+## Rotating the keys
+
+For an existing account you can rotate the private key, public key pair, i.e., use a new pair of private, public keys at regular intervals. To rotate the keys you must pass the current authentication key of your account. This will generate and store a new authentication key in the account. **However, the account address will remain unchanged.**
+
+## Access control with signer
+
+The sender of a transaction is represented by a signer. When a `signer` is specified in a function in a transaction, then the `signer` is the only entity capable of adding or removing resources into an account. See the below Move example code with `&signer` in the `initialize` and `withdraw` functions. When a `signer` is not specified in a function, for example, the below `deposit` function, then no access controls exist for this function:
```

Comment:
```suggestion
The sender of a transaction is represented by a signer. When a `signer` is specified in a Move module function, the Move VM will assert that the `signer` of the transaction that triggered that function is the owner of the account. See the below Move example code with `&signer` in the `initialize` and `withdraw` functions. When a `signer` is not specified in a function, for example, the below `deposit` function, then no access controls exist for this function:
```

Reply:
I deliberately avoided introducing any Move/MoveVM related terms in this intro doc. When I read the above suggestions, I strongly felt I am jumping too ahead of myself by talking about assertions, transactions triggering functions, VM and Move etc. I will table this to cover in a separate doc (I got lots of docs in my head I should write).

Reply:
We don't have to mention the move VM or use the word "assertion", we could just say "ensure", but I think it's important to phrase it that something ensures that only the owner of the account can run functions that take in the signer. Mentioning adding / removing resources, while simpler, is not actually correct, the check is more general than that. I think as it is now it sells our security model short.

Reply:
Can you suggest a phrasing here?

Reply:
How's this, @banool? A slight edit of your suggestion. "The sender of a transaction is represented by a signer. When a `signer` is specified in a Move module function in a transaction, then the Move VM in the Aptos blockchain will assert that the `signer` of the transaction (that triggered that function) is the owner of the account. See the below Move example code with `signer` in the `initialize` and `withdraw` functions. When a `signer` is not specified in a function, for example, the below `deposit` function, then no access controls exist for this function:"

Reply:
That sounds great, thanks for clarifying this piece.